goshopify.go

```diff
@@ -222,6 +225,10 @@ func (c *Client) WithLogger(l Logger) *Client {
 // response. It does not make much sense to call Do without a prepared
 // interface instance.
 func (c *Client) Do(req *http.Request, v interface{}) error {
+	// Self-throttle if bucket capacity is specified and exceeds threshold
+	if c.app.BucketThreshold > 0 && bucketCapacity > c.app.BucketThreshold {
```

Comment:
Where are  `c.app.BucketThreshold` and `c.app.HeavyLoadDelay` set?

Reply:
in the go-shopify App set in the callers. you'll see in the next PR